### PR TITLE
Fix undefined props in README examples

### PR DESCRIPTION
**Pull request by aunyks**
In the example code snippets in the README, some components reference their given props. But, no props are given, so these prop values are undefined. The specific prop value that is missing from the *Application* and *FlashingLed* components is the "interval" prop.

In this commit, I provide the interval prop to the Application component in the first example at the time that it's rendered, and I gave the interval prop a default value in the FlashingLed component to prevent unnecessary repetition in the second example.

TLDR: I gave *Application* and *FlashingLed* an interval prop value, because it'd be undefined otherwise.

### CODE REVIEW

**Comment by aunyks** [comment]
The changes were only made to README, so I'm not sure why the build failed...

**Comment by iamdustan** [comment]
Thanks @aunyks. Don’t worry about the CI. Current master didn’t constrain it’s React version tight enough for eating internals :smile:
